### ISSUE TITLE
diyabc-rf GUI does not load

### ISSUE THREAD

**Issue by RoseiYu-0321** [diyabc/diyabcGUI#121]
I have downloaded the DIYABC-RF GUI at https://github.com/diyabc/diyabcGUI/releases/latest and unzip the fodlder.
By double-clicking "DIYABC-RF_GUI.bat", the program quickly loads and then disappears.
Don't know what's wrong with that. Any help would be appreciated. Thanks.

**Comment by Jinyu8579** [comment]
Have you ever make DIYABC work? I encounter the same problem with you.

**Comment by RoseiYu-0321** [comment]
No. It doesn't work. I couldn't use it even in the R. There is some problem.

**Comment by gdurif** [comment]
Hi,
Sorry for the late response, I am under quite a rush for the new year. I'll look into it and get back to you ASAP.
Best,

**Comment by gdurif** [comment]
Hi,
Could you give me more details? Which OS you are using?
Thanks
Best

**Comment by gdurif** [comment]
FYI, we just published a new release (version [1.2.1](https://github.com/diyabc/diyabcGUI/releases/tag/v1.2.1-beta)) which should fix several issues. Could you tell me if it works better?
Thanks in advance,
Best regards

**Comment by gdurif** [comment]
@AntMacDF sorry for the trouble. Could you tell me which version you are using on Windows ? the Windows release or the R package ?

If using the Windows release, is there a console window that opens when you try to run the app ?

In addition, you should be able to find a log file in `C:\Users\<username>\AppData\Local\Temp` (either directly there or in a subdirectory). If so, could you copy it here ?

Thanks

**Comment by AntMacDF** [comment]
Thank you.
I'm using the Windows release.
Yes, a console window opens, but very quickly. Please find in attachment a print screen.
And the log file seems not present; C:\Users\<username>\AppData\Local\Temp doesn't exist

Best

<img width="784" alt="Capture d’écran 2022-03-01 à 10 12 12" src="https://user-images.githubusercontent.com/100463341/156139926-f27df22b-a002-43e3-9fae-ff8e22198f0d.png">
.

**Comment by gdurif** [comment]
Thank you very much, I understand the problem now. I will fix it today. Thanks again

**Comment by gdurif** [comment]
@AntMacDF I have another question. I am not able to replicate the issue. Could you tell me where (in your file system) did you put the release directory ?

**Comment by AntMacDF** [comment]
Sorry, definitely a problem during extraction. I downloaded the zip file (DIYABC-RF_GUI_1.2.1_windows.zip) again, extracted it again to my desktop and it boots.
Sorry for the time spent and thank you for your reactivity.
Any idea when a version for MacOS will be available?

Best

**Comment by gdurif** [comment]
No problem. I am glad we found a solution.

Regarding MacOS, we are working on it and I have no timeline at the moment. You can check #19 to follow the progress.

In the mean time, you could use the R package. You would need to install R, and then follow the instructions here: https://github.com/diyabc/diyabcGUI#r-package-installation
